components/server/ee/src/prebuilds/bitbucket-app.ts

```diff
@@ -89,15 +93,47 @@ export class BitbucketApp {
                 return undefined;
             }
 
-            console.log('Starting prebuild.', { contextURL })
-            // todo@alex: add branch and project args
-            const ws = await this.prebuildManager.startPrebuild({ span }, { user, contextURL, cloneURL: data.gitCloneUrl, commit: data.commitHash});
+            console.debug('Bitbucket push event: Starting prebuild.', { contextURL });
+
+            const projectOwner = await this.findProjectOwner(data.gitCloneUrl, user);
+            const ws = await this.prebuildManager.startPrebuild({ span }, {
+                user: projectOwner?.user || user,
+                project: projectOwner?.project,
+                branch: data.branchName,
+                contextURL,
+                cloneURL: data.gitCloneUrl,
+                commit: data.commitHash
+            });
             return ws;
         } finally {
             span.finish();
         }
     }
 
+    protected async findProjectOwner(cloneURL: string, webhookInstaller: User): Promise<{ user: User, project?: Project } | undefined> {
```

Comment:
@AlexTugarev IMO we need a comment here to explain why we need to find the project owner, and how we get there (project.userId, teamId, members with "owner" role).